src/store/slices/blogsSlice.js

```diff
@@ -2,40 +2,21 @@ import { createSlice } from "@reduxjs/toolkit";
 
 const blogsSlice = createSlice({
     name: 'blogs',
-    initialState : [
-        {
-            id: 0,
-            username: '',
-            question: '',
-            text: '',
-            upvotes: 0,
-            downvotes: 0,
-            date: '',
-            comments: [
-                {
-                    id: 0,
-                    comUsername: '',
-                    comment: '',
-                    upvotes: 0,
-                    downvotes: 0,
-                    time: ''
-                }
-            ]
-        }
-    ],
+    initialState : {
+    },
     reducers: {
         addBlog(state,action){
             // assumption:
-            // action.payload === {blog object}
-            state.push(action.payload);
+            // action.payload === [blogs array]
```

Comment:
Also, need to compliment your comment adding routines. I am much more experienced than you and I rarely do this. I should do this too. :)